src/types/package.rs

```diff
@@ -112,23 +124,29 @@ impl FromStr for PackageType {
     type Err = ();
 
     fn from_str(input: &str) -> Result<Self, Self::Err> {
-        match input.to_lowercase().as_str() {
-            "npm" => Ok(Self::Npm),
-            "python" | "pypi" => Ok(Self::PyPi),
-            "maven" | "maven-central" => Ok(Self::Maven),
-            "ruby" | "rubygems" | "gem" => Ok(Self::RubyGems),
-            "nuget" | "dotnet" => Ok(Self::Nuget),
-            "cargo" => Ok(Self::Cargo),
-            "golang" => Ok(Self::Golang),
-            _ => Err(()),
-        }
+        Ok(match input {
```

Comment:
I'd assume a match statement is used here for the sake of formatting? Generally the match doesn't really do anything so the "canonical" representation would be an if/else chain, but I'm not too bothered by it in this case. However the match part is completely useless.

Reply:
Yeah it behaves like if/else. I think it looks slightly better as a match because I'm conceptually matching the input string to an output enum, but it's about the same either way.